Patches/turk's guns/Patch_Guns.xml

```diff
@@ -0,0 +1,317 @@
+		 <value>
+		  <tools>
+		   <li Class="CombatExtended.ToolCE">
+		    <label>grip</label>
+			<capacities>
+			 <li>Blunt</li>
+			</capacities>
+			<power>9</power>
+			<cooldownTime>2</cooldownTime>
+			<armorPenetrationBlunt>0.5</armorPenetrationBlunt>
+			<linkedBodyPartsGroup>Grip</linkedBodyPartsGroup>
+		   </li>
+		  </tools>
+		 </value>
+		</li>
+		
+		<li Class="PatchOperationReplace">
+		 <xpath>Defs/ThingDef[defName="Gun_MX21FourSmol"]/tools</xpath>
+		 <value>
+		  <tools>
+		   <li Class="CombatExtended.ToolCE">
+		    <label>barrels</label>
+			<capacities>
+			 <li>Blunt</li>
+			</capacities>
+			<power>12</power>
+			<cooldownTime>2</cooldownTime>
+			<armorPenetrationBlunt>0.5</armorPenetrationBlunt>
+			<linkedBodyPartsGroup>Barrels</linkedBodyPartsGroup>
+		   </li>
+		  </tools>
+		 </value>
+		</li>
+		
+		<li Class="PatchOperationReplace">
+		 <xpath>Defs/ThingDef[defName="Gun_MX21Four"]/tools</xpath>
+		 <value>
+		  <tools>
+		   <li Class="CombatExtended.ToolCE">
+		    <label>barrels</label>
+			<capacities>
+			 <li>Blunt</li>
+			</capacities>
+			<power>12</power>
+			<cooldownTime>2.1</cooldownTime>
+			<armorPenetrationBlunt>0.55</armorPenetrationBlunt>
+			<linkedBodyPartsGroup>Barrels</linkedBodyPartsGroup>
+		   </li>
+		  </tools>
+		 </value>
+		</li>
+		
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+		  <defName>Gun_AATwelve</defName>
+		  <statBases>
+		    <WorkToMake>31500</WorkToMake>
+		    <SightsEfficiency>1.0</SightsEfficiency>
+			<ShotSpread>0.14</ShotSpread>
+			<SwayFactor>1.49</SwayFactor>
+			<Bulk>10.66</Bulk>
+			<Mass>5.20</Mass>
+			<RangedWeapon_Cooldown>0.38</RangedWeapon_Cooldown>
+		  </statBases>
+		  <costList>
+		  <Steel>60</Steel>
+		  <Chemfuel>10</Chemfuel>
+		  <ComponentIndustrial>5</ComponentIndustrial>
+		  </costList>
+		  <Properties>
+		    <recoilAmount>2.26</recoilAmount>
+		    <verbClass>CombatExtended.Verb_ShootCE</verbClass>
+			<hasStandardCommand>true</hasStandardCommand>
+			<defaultProjectile>Bullet_12Gauge_Buck</defaultProjectile>
+			<burstShotCount>3</burstShotCount>
+			<ticksBetweenBurstShots>12</ticksBetweenBurstShots>
+			<warmupTime>0.6</warmupTime>
+			<range>20</range>
+			<soundCast>Shot_Shotgun</soundCast>
+			<soundCastTail>GunTail_Heavy</soundCastTail>
+			<muzzleFlashScale>9</muzzleFlashScale>
+		  </Properties>
+		  <AmmoUser>
+		    <magazineSize>32</magazineSize>
+			<reloadTime>4.9</reloadTime>
+			<ammoSet>AmmoSet_12Gauge</ammoSet>
+		  </AmmoUser>
+		  <FireModes>
+		    <aiAimMode>Snapshot</aiAimMode>
+            <aiUseBurstMode>true</aiUseBurstMode>
+            <aimedBurstShotCount>2</aimedBurstShotCount>
+		  </FireModes>
+		  <weaponTags>
+		    <li>IndustrialGunAdvanced</li>
+		  </weaponTags>
+		  <researchPrerequisite>GasOperation</researchPrerequisite>
+		</li>
+
+		<li Class="CombatExtended.PatchOperationMakeGunCECompatible">
+		  <defName>Gun_HAMR</defName>
```

Comment:
I'd definitely patch this gun as a SCAR H (7.62 NATO) with a larger magazine. The SCAR H is patched already in the files, so it should be a pretty easy swap.

Reply:
I would keep the gun as is to fill the role of SAW but let me know if you insist on this